ui/plugins/auto_classification/errorLine.html

```diff
@@ -1,131 +0,0 @@
-<div ng-class="['status', status]">&nbsp;</div>
-
-<div ng-if="line.verified">
-  <span ng-if="!line.verifiedIgnore"
-        class="badge badge-xs badge-primary"
-        title="This line is verified">Verified</span>
-
-  <span ng-if="line.verifiedIgnore"
-        class="badge badge-xs badge-ignored"
-        title="This line is ignored">Ignored</span>
-</div>
-
-<div class="classification-line-detail">
-  <div ng-if="failureLine">
-    <span ng-if="failureLine.action === 'test_result'">
-      <span ng-class="{'ignored-line': line.verifiedIgnore}">
-        <strong class="failure-line-status">{{ ::failureLine.status }}</strong>
-        <span ng-if="failureLine.expected != 'PASS' && failureLine.expected != 'OK'">
-          (expected <strong>{{ ::failureLine.expected }}</strong>)
-        </span>
-        | <strong>{{ ::failureLine.test }}</strong>
-        <span ng-if="failureLine.subtest">| {{ ::failureLine.subtest }}</span>
-      </span>
-      <div ng-if="failureLine.message && !line.verifiedIgnore"
-           ng-init="messageExpanded=false"
-           class="failure-line-message">
-        <span class="failure-line-message-toggle fa fa-fw fa-lg"
-              ng-class="{'fa-caret-down': messageExpanded, 'fa-caret-right': !messageExpanded}"
-              ng-click="messageExpanded = !messageExpanded"></span>
-        <span ng-if="!messageExpanded"
-              class="failure-line-message-collapsed">{{ ::failureLine.message }}</span>
-        <span ng-if="messageExpanded"
-              class="failure-line-message-expanded">{{ ::failureLine.message }}</span>
-      </div>
-    </span>
-    <span ng-if="::failureLine.action === 'log'">
-      LOG {{ ::failureLine.level }} | {{ ::failureLine.message }}
-    </span>
-    <span ng-if="::failureLine.action === 'crash'">
-      CRASH |
-      <span ng-if="::failureLine.test">
-        <strong>{{ ::failureLine.test }}</strong> |
-      </span>
-      application crashed [{{ ::failureLine.signature }}]
-    </span>
-    <span>[<a title="Open the log viewer in a new window"
-       target="_blank" rel="noopener"
-       href="{{ logUrl }}"
-       class="">log…</a>]</span>
-  </div>
-  <div ng-if="!failureLine">
-    <span ng-bind-html="::searchLine | escapeHTML | highlightLogLine"></span>
```

Comment:
I think the `escapeHTML` filter in `ui/js/filters.js` is now unused (the test in `filter.tests.js` will need adjusting to use the function in `displayHelper.js` or else be removed)